<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1590</width>
    <height>913</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_Secuencia1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Secuencia2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Secuencia 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Secuencia 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ejemplo:{5,7,*9,3,4,6,1} o {6,8,1,2,4,8*,9,7}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>1071</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Intrucciones: Ingresar la secuenciaen los cuadros de texto, tal y como lo indica el ejemplo, el origen lo puede indicar con un asterisco</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Audio1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Audio2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RutaAudio1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RutaAudio2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Calcular">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="title">
    <string>Operaciones</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>k =</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_n0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_KDiezm">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Diezmacion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Diezmacion</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Convolucion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Convolucion</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_KInterp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Interpolacion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolacion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>n0 =</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Multiplicacion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiplicacion</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Suma">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Suma</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Reflexion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>k =</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Desplazamiento">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Desplazamiento</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Resta">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resta</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Interpolaciones">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A Cero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Escalon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lineal</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>510</y>
     <width>501</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Sec2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>80</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Sec1">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>1001</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Respuesta">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_Reflexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit_Secuencia2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>948</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Desplazamiento</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit_Secuencia2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>981</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Diezmacion</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit_Secuencia2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>964</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Interpolacion</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit_Secuencia2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>992</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Reflexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_Audio2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Desplazamiento</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_Audio2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1045</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Diezmacion</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_Audio2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1019</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Interpolacion</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_Audio2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
